src/components/Navbar/NavItems.tsx

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable @typescript-eslint/no-empty-function */
+import React from 'react';
+import { Divider } from '@material-ui/core';
+import { User, FileText, Folder, Type, MessageCircle } from 'react-feather';
+import { NavItem, NavItemProps } from '../NavItem';
+import { CFC } from '../../typings/components';
+
+type NavItemsProps = {
+  onClick?: NavItemProps['onClick'];
+};
+
+const NavItems: CFC<NavItemsProps> = ({ onClick }) => {
+  return (
+    <>
+      <NavItem to="/about" onClick={onClick} icon={User}>
+        About
+      </NavItem>
+      <Divider flexItem orientation="vertical" />
+      <NavItem to="/resume" onClick={onClick} icon={FileText}>
```

Comment:
currently named `resume-page` so we should change the route or page name :D

Reply:
I think, there is no need to add a page suffix, but I'm open if you see any benefits of such an approach

Reply:
Not really, I just named it like this and everyone went with a trend 😄  I'd totally change it